Source/Extensions/String+CamelCase.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  String+CamelCase.swift
+//  Github Notifications
+//
+//  Created by Chananchida F. on 9/24/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+
+    var uppercasingFirst: String {
+        return prefix(1).uppercased() + dropFirst()
+    }
+
+    var lowercasingFirst: String {
+        return prefix(1).lowercased() + dropFirst()
+    }
+
+    var camelized: String {
+        guard !isEmpty else { return "" }
+        let parts = components(separatedBy: CharacterSet.alphanumerics.inverted)
+        let first = String(describing: parts.first!).lowercasingFirst
```

Comment:
Let's use optional instead or you can use `guard` too
```suggestion
        let parts = components(separatedBy: CharacterSet.alphanumerics.inverted)
        let first = String(describing: parts.first ?? "").lowercasingFirst
```